__tests__/app.test.js

```diff
@@ -88,6 +88,22 @@ describe("/api/articles", () => {
             expect(body.articles).toBeSortedBy("created_at")
         ))
     })
+    test("GET: 200 - take a sort_by query and respond with articles sorted by given column name", () => {
+        return request(app)
+        .get("/api/articles?sort_by=author")
+        .expect(200)
+        .then(({body})=>(
+            expect(body.articles).toBeSortedBy("author")
+        ))
+    })
+    test("GET: 400 - returns an error when given a non-valid sort_by", () => {
+        return request(app)
+        .get("/api/articles?sort_by=invalidColumn")
+        .expect(400)
+        .then(({ body: { msg } }) => {
+            expect(msg).toBe("Bad Request")          
+        })
+    })
```

Comment:
two behaviours untested here:

- order value not found (eg not ASC or DESC)
- order or sort_by could be misspelled